### PR TITLE
Replace `colors` with `chalk`

### LINKED ISSUE #83: Replace colors with chalk
Because extending String prototype is uncool.

### PR DESCRIPTION
**Pull request by yeskunall**
See [chalk#origin-story](https://github.com/chalk/chalk\#origin-story) for explanation.

<hr />

Closes #83.

### CODE REVIEW

File: bin/airtap.js [hunk 1]
```diff
@@ -65,35 +65,16 @@ for (var key in config) {
...
-      console.error('Unable to get available browsers for saucelabs'.red)
-      console.error(err.stack)
+      console.error(chalk.bold.red('Unable to get available browsers for saucelabs'))
+      console.error(chalk.red(err.stack))
       return process.exit(1)
     }
 
     for (var browser in allBrowsers) {
+      // TODO: Debug log?
       console.log(browser)
```
Comment (yeskunall) [inline bin/airtap.js]: Is this intentional? 😅

File: bin/airtap.js [hunk 2]
```diff
@@ -65,35 +65,16 @@ for (var key in config) {
...
     if (err) {
-      console.error('Unable to get available browsers for saucelabs'.red)
-      console.error(err.stack)
+      console.error(chalk.bold.red('Unable to get available browsers for saucelabs'))
+      console.error(chalk.red(err.stack))
       return process.exit(1)
     }
 
     for (var browser in allBrowsers) {
+      // TODO: Debug log?
```
Comment (yeskunall) [inline bin/airtap.js]: Is this intentional? 😅

Comment (yeskunall) [inline bin/airtap.js]: Is this intentional? 😅

**Comment by ralphtheninja** [comment]
LGTM

File: bin/airtap.js (hunk 2 above)
Comment (ralphtheninja) [reply bin/airtap.js]: No idea.

Comment (yeskunall) [reply bin/airtap.js]: I'll keep it.

**Comment by yeskunall** [comment]
I'll go ahead and merge this. Please ping me if there's anything you wanna know related to this PR 🤗

**Pull request merged**